website/content/api-docs/hcp-link.mdx

```diff
@@ -0,0 +1,186 @@
+---
+layout: api
+page_title: HCP Linking HTTP API
+description: The Link resource allows for linking your cluster to HCP Consul Central.
+---
+
+# Link HTTP API
+
+-> **1.18.0+:** The Link API is available in Consul versions 1.18.0 and newer.
+
+## Establish or Update Link to HCP Consul Central
+
+This endpoint creates or updates a Link, which establishes a connection with HCP Consul Central.
+
+| Method | Path                          | Produces           |
+| ------ | ----------------------------- | ------------------ |
+| `PUT`  | `/api/hcp/v2/link/global`     | `application/json` |
+
+The table below shows this endpoint's support for
+[blocking queries](/consul/api-docs/features/blocking),
+[consistency modes](/consul/api-docs/features/consistency),
+[agent caching](/consul/api-docs/features/caching), and
+[required ACLs](/consul/api-docs/api-structure#authentication).
+
+| Blocking Queries | Consistency Modes     | Agent Caching | ACL Required                   |
+| ---------------- | --------------------- | ------------- | ------------------------------ |
+| `NO`             | `stale`, `consistent` | `none`        | `operator:write`, `acl:write`  |
```

Comment:
I had a question about this table (and the similar ones below) that I asked in a [thread here](https://hashicorp.slack.com/archives/C04AM35BFQT/p1705961384480169). This is my best guess at filling out this table but not sure if it's correct.